lib/features/local_live_log.js

```diff
@@ -1,9 +1,3 @@
-/**
```

Comment:
We should probably not only remove this, but replace it with documentation about what this module/feature does.